extensions/amp-story/0.1/page-advancement.js

```diff
@@ -222,16 +222,20 @@ class ManualAdvancement extends AdvancementConfig {
    * @param {!Element} element The element that, when clicked, can cause
    *     advancing to the next page or going back to the previous.
    */
-  constructor(element) {
+  constructor(autoAdvanceStr, element) {
     super();
     this.element_ = element;
+    this.autoAdvanceStr_ = autoAdvanceStr;
     this.clickListener_ = this.maybePerformNavigation_.bind(this);
   }
 
   /** @override */
   start() {
     super.start();
     this.element_.addEventListener('click', this.clickListener_, true);
+    if (!this.autoAdvanceStr_) {
+      this.onProgressUpdate();
+    }
   }
```

Comment:
The last page's `start()` method gets called when the replay button is hit on the bookend, need to investigate why.